src/main/java/net/adoptium/UserSystem.java

```diff
@@ -0,0 +1,41 @@
+package net.adoptium;
+
+import net.adoptopenjdk.api.v3.models.Architecture;
+import net.adoptopenjdk.api.v3.models.OperatingSystem;
+
+public class UserSystem {
+    private OperatingSystem os;
+    private Architecture arch;
+
+    public UserSystem() {
+
+    }
+
+    public UserSystem(OperatingSystem os, Architecture arch) {
```

Comment:
Do we really need several constructors and setters?

Reply:
removed unused Constructor